niceman/config.py

```diff
@@ -126,7 +129,7 @@ def _get_default_file_candidates(self):
         # XDG user config
         home_cfg_base_path = self.dirs.user_config_dir
         if os.path.isabs(home_cfg_base_path):
-            cfg_file_candidates.append(opj(home_cfg_base_path, 'niceman.cfg'))
+            cfg_file_candidates.append(opj(home_cfg_base_path, 'config'))
```

Comment:
while at it -- may be it should be unified? i.e. as it was in the code here and just docs adjusted?

Reply:
Sorry, I don't understand your comment.

How could it be unified by just adjusting the docs?  If we're going to unify to niceman.cfg, wouldn't $XDG_CONFIG_DIRS/niceman/config and $PWD/.niceman/config need to be changed?

Also, this change is a separate commit, so feel free to just revert 37cdc86 if you want to stay with $XDG_CONFIG_HOME/niceman/niceman.cfg.

Reply:
yes, sorry -- missed those as well.  But what do you think -  worth having them consisent?

Reply:
Hmm, yeah, I'm fine either way.  NAME/config looks right to me because of things like ".config/git/config", but I see the point of "let's just use one name".

My only strong preference is that we use the same pattern for $XDG_CONFIG_HOME, $XDG_CONFIG_DIRS, and $PWD/.niceman.

Reply:
OK, well since you seem to prefer niceman.cfg and I have no preference and others haven't weighed in, I've changed all the locations to use niceman.cfg.